pkg/multitenant/tenantcostmodel/settings.go

```diff
@@ -118,41 +134,261 @@ var (
 		PgwireEgressCostPerMiB,
 		ExternalIOEgressCostPerMiB,
 		ExternalIOIngressCostPerMiB,
+		RegionalCostMultiplierTableSetting,
 	}
 )
 
+func validateRegionalCostMultiplierTableSetting(values *settings.Values, tableStr string) error {
+	_, err := parseRegionalCostMultiplierTableSetting(tableStr)
+	return err
+}
+
+func parseRegionalCostMultiplierTableSetting(
+	tableStr string,
+) (*RegionalCostMultiplierCompactTable, error) {
+	var compact RegionalCostMultiplierCompactTable
+	if tableStr == "" {
+		return &compact, nil
+	}
+	if err := json.Unmarshal([]byte(tableStr), &compact); err != nil {
+		return nil, err
+	}
+	if err := compact.Validate(); err != nil {
+		return nil, err
+	}
+	return &compact, nil
+}
+
+// RegionalCostMultiplierCompactTable describes the cost multiplier table for
+// cross-region transfers in compacted form. This model assumes that the cost
+// is the same for both directions.
+type RegionalCostMultiplierCompactTable struct {
+	// Regions represents the list of region names. The indexes of the regions
+	// will be used alongside with the matrix.
+	Regions []string `json:",omitempty"`
```

Comment:
nit: I think we should use snake_case for the json keys.

CRDB isn't completely consistent with json style, but greping through the code base suggests that snake case is the dominant json naming convention.

The only other setting that accepts json is https://github.com/cockroachdb/cockroach/blob/fefa5fe16cc38c14a1406f6c91002493faa1cfb4/pkg/ccl/oidcccl/settings.go#L104 and it uses snake case as well.